akka-stream/src/main/scala/akka/stream/javadsl/Compression.scala

```diff
@@ -34,6 +34,14 @@ object Compression {
     scaladsl.Compression.gzip.asJava
 
   /**
+   * Same as [[gzip]] with a custom level.
+   *
+   * @param level Compression level (0-9)
+   */
+  def gzip(level: Int): Flow[ByteString, ByteString, NotUsed] =
```

Comment:
defaults here

Reply:
Oops, not in the Java version, of course ;)

Reply:
That reminded me I forgot the deflate java api :)